CRM/PCP/Page/PCPInfo.php

```diff
@@ -57,6 +57,7 @@ public function run() {
 
     CRM_Utils_System::setTitle($pcpInfo['title']);
     $this->assign('pcp', $pcpInfo);
+    $this->assign('currency', $pcpInfo['currency']);
```

Comment:
I could use pcp.currency directly on the template but for consistency with *all* other crmMoney modifiers I'm assigning currency directly to the template.